packages/core/src/errors.ts

```diff
@@ -427,3 +441,23 @@ export class Verification extends AuthError {
 export class MissingCSRF extends SignInError {
   static type = "MissingCSRF"
 }
+
+// TODO: Review this list to make sure it's complete and we only send non-hinting errors to the client.
+const clientErrors = new Set<ErrorType>([
```

Comment:
WDYT if we introduce a type `ClientErrorType` that contains AuthError the list and then above in `ErrorType` we can do:
```ts
type ErrorType = ServerConfigurationErrorType | ClientErrorType | SensitiveErrorType
type ServerConfigurationErrorType = 
  | "MissingAdapter"
  | "MissingAdapterMethods"
  | "MissingAuthorize"
  | "MissingSecret"
  // so on...
```


and then we can go through the smaller lists easier

Reply:
I did not want to many types to maintain, but maybe a `ClientErrorType` is a good idea!

Reply:
Let's just go ahead without it then, we already have this set which should be enough